test/java/seedu/addressbook/commands/ViewAllCommandTest.java

```diff
@@ -0,0 +1,125 @@
+package seedu.addressbook.commands;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import static seedu.addressbook.util.TestUtil.assertCommandResult;
+
+public class ViewAllCommandTest {
+    private AddressBook addressBook;
+    private AddressBook emptyAddressBook;
+    private List<ReadOnlyPerson> emptyDisplayList;
+    private List<ReadOnlyPerson> listWithAll;
+    private List<ReadOnlyPerson> listWithSome;
+
+    @Before
+    public void setUp() throws Exception {
+        TypicalPersons td = new TypicalPersons();
+
+        emptyAddressBook = TestUtil.createAddressBook();
```

Comment:
@damithc This `TestUtil.createAddressBook()` is throwing an excpetion in `TestUtil `Class so cannot put outside right?